syntax = "proto3";

message StartDriverRequest {
    string udid = 1;
}
message StartDriverResponse {}

message InstallAppRequest {
    string udid = 1;
}
message InstallAppResponse {}

message UninstallAppRequest {
    string udid = 1;
}
message UninstallAppResponse {}

message ExecuteTestRequest {}
message ExecuteTestResponse {}

message MobiliumMessage {
    oneof message {
        StartDriverRequest startDriverRequest = 1;
        StartDriverResponse startDriverResponse = 2;
        InstallAppRequest installAppRequest = 3;
        InstallAppResponse installAppResponse = 4;
        UninstallAppRequest uninstallAppRequest = 5;
        UninstallAppResponse uninstallAppResponse = 6;
        ExecuteTestRequest executeTestRequest = 7;
        ExecuteTestResponse executeTestResponse = 8;
    }
}
